client/styles/base.styl

```diff
@@ -163,5 +163,6 @@ span.error
   display block
   color uber-orange
 
-section.workflow-list
-  overflow-y: auto
+section.workflow-list {
+  overflow-y: hidden;
+}
```

Comment:
different structure/syntax than elsewhere - is that relevant?

Reply:
Stylus allows both syntax. I don't like the bracketless style because it can lead to confusion when nesting CSS (which has its own problems anyway). However it is mostly dominant in the codebase. Perhaps there is a linter that we could use to enforce using brackets? I don't mind changing either way so let me know and I can make this change in a different PR.

Reply:
Looks like stylelint supports `.vue` files
https://stylelint.io/